ui/helpers/urlHelper.js

```diff
@@ -90,8 +90,8 @@ export const getJobSearchStrHref = function getJobSearchStrHref(jobSearchStr) {
 };
 
 //TH domain is needed here because Intermittent Failures View currently uses hash router
-export const jobsUrl = function getJobsUrl(tree, revision) {
-  return `https://treeherder.mozilla.org/#/jobs?repo=${tree}&revision=${revision}`;
+export const jobsUrl = function getJobsUrl(tree, revision, jobId) {
+  return `https://treeherder.mozilla.org/#/jobs?repo=${tree}&revision=${revision}&selectedJob=${jobId}`;
```

Comment:
This works to use the same domain as wherever it's running if you use:

    return `/#/jobs?repo=${tree}&revision=${revision}&selectedJob=${jobId}`

Then it'll pickup ``localhost:5000`` or stage/prototype, if that's where you are.  I think this is preferable.  Sorry I missed this in an earlier review.  :)

Reply:
@camd I saw the hash router making that url a route of intermittents-failure.html (intermittent-failures.html#/jobs...) when we were testing it on prototype, but I realize now it's because I had forgotten the / before the hash. I'll remove the domain and resubmit shortly :)